adapters/networkHandlerFactory.js

```diff
@@ -19,6 +19,9 @@ const {
 const {
   networkHandler: GA4NetworkHandler
 } = require("../v0/destinations/ga4/utils");
+const {
+  networkHandler: YahooDSP
+} = require("../v0/destinations/yahoo_dsp/util");
 
```

Comment:
Please remove this as we are not using transformerProxy.

Reply:
It's needed. We'll are releasing the feature soon.
CC @sanpj2292

Reply:
Okay, we can keep it. But we are not following this for any other new integrations. Is there any plan regarding this that we can follow for all new integrations?